grails-app/domain/org/pih/warehouse/core/ProductPrice.groovy

```diff
@@ -0,0 +1,65 @@
+/**
+* Copyright (c) 2012 Partners In Health.  All rights reserved.
+* The use and distribution terms for this software are covered by the
+* Eclipse Public License 1.0 (http://opensource.org/licenses/eclipse-1.0.php)
+* which can be found in the file epl-v10.html at the root of this distribution.
+* By using this software in any fashion, you are agreeing to be bound by
+* the terms of this license.
+* You must not remove this notice, or any other, from this software.
+**/
+package org.pih.warehouse.core
+
+import org.pih.warehouse.auth.AuthService
+import org.pih.warehouse.product.ProductPackage
+
+class ProductPrice implements Serializable {
```

Comment:
Ok, I think I figured out how to handle what was bothering me about my original design. As a standalone class this has nothing to do with product so it should not be named ProductPrice. The best name I've seen for a class like this is PriceComponent. However, I think PriceComponent takes us down a path we don't want to go right now we'll leave it as ProductPrice for now.